packages/client/lib/client/commands.ts

```diff
@@ -358,5 +359,7 @@ export default {
     UNWATCH,
     unwatch: UNWATCH,
     WAIT,
-    wait: WAIT
+    wait: WAIT,
+    WAITAOF,
+    waitaof: WAITAOF
```

Comment:
Change to `waitAOF` or `waitAof` (I'm actually not sure which one we should go with :P)